<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameTextCreditsTab</class>
 <widget class="QWidget" name="GameTextCreditsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLineEdit" name="bossCredits_lineEdit3"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bossCredits_lineEdit1"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="bossCredits_lineEdit2"/>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="stageSelect_comboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bosses Credits:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Concept/Idea:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Design:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QTextEdit" name="gameCredits_textEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Game Credits:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
